### PR TITLE
Patrykoss

### PR DESCRIPTION
**Pull request by Patrykoss**
Without a consistent interface(css)

### CODE REVIEW

File: Client/Components/AnswerTextComponent.razor [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+        </div>
+        <div>
+            <button @onclick="@(()=> Discard.InvokeAsync())">Discard</button>
+        </div>
+    </div>
+</div>
+
+
+@code {
+    [Parameter]
```
Comment (Ventan00) [inline Client/Components/AnswerTextComponent.razor:17]: missing docblock

**Review by pnarozniak — APPROVED** [review]
Ok

File: Client/Repositories/Implementations/AnswerRepository.cs [hunk 2]
```diff
@@ -16,9 +18,59 @@ public AnswerRepository(IHttpService httpService)
             this.httpService = httpService;
         }
```
Comment (Ventan00) [inline Client/Repositories/Implementations/AnswerRepository.cs:19]: should be _httpService

**Review by Ventan00 — CHANGES_REQUESTED** [review]
Docblocks are part of the next update, but if you implement them in a file in one place, do it throughout the file. Also, when using private variables, use `_name` instead of `name`

**Review by Ventan00 — APPROVED** [review]
Ok!

**Pull request merged**